### ISSUE TITLE
Reduce the build size

### ISSUE THREAD

**Issue by jbphet** [phetsims/number-line-integers#26]
I just built and published version 1.0.0-dev.6, and it is 12M in size (measured using the `du -sh` command in GitBash).  This is big.  For a point of comparison, I build Gas Properties recently, and it measured 1.9M.  I think the main causes of the large size are the 12 map images and the temperature data that goes with them.  Using some different compression algorithms on the maps may help.  Also, compressing the temperature data or using Celsius instead of Kelvin will likely also reduce the size.  I think we should start by attempting those items, then see where we're at and if further effort is required.

**Comment by SaurabhTotey** [comment]
So far, when I run `du -sh` on the build folder, I get a size of 20M whereas I was getting a size of 39M before. For comparison, I am getting a size of 21M on gas-properties.

**Comment by amanda-phet** [comment]
@SaurabhTotey and I did experiment a bit with images and found a way to compress the map images. I haven't done it for all 12 months, but if we decide to go that route please assign this issue to me to deliver those new assets.

**Comment by jbphet** [comment]
@amanda-phet - let's try the compressed images.

**Comment by amanda-phet** [comment]
I've uploaded the new images, but I think you'll need to point to them in the code.

**Comment by jbphet** [comment]
I've updated the code and removed the old .png files.  I also fixed the file names (they had the word "monthy" instead of "monthly" in them).  I did an before-and-after comparison and the built sim file (` number-line-integers_en_phet.html`) went from 5628kB to 3992kB.  Perhaps this is enough.  I'll compare to other sims and decide.

**Comment by jbphet** [comment]
I had a bit of a discussion on Slack with other devs.  @pixelzoom pointed out that Gas Properties, which doesn't have many images, weighs in at ~2MB and EFAC, which has quite a few images, is 3.7MB.  It seems like this sim is in that ballpark.

@ariel-phet - is this size (basically 4MB) okay?  Further reduction is likely to be challenging, so I thought I'd check in as to whether we're "there yet".

**Comment by pixelzoom** [comment]
Taking a quick look at images/. Your home and nav icons could be created programmatically.  And many of your non-`world-temp*` images seems to be much bigger than necessary.  Anything bigger than 1.5x what you need is liable to end up with with scaling artifacts.

world-temperatures-monthy-averaged-2018-01.jpg and world-temperatures-monthy-averaged-2018-01.png appear to be duplicates.  Why do you need both formats?

**Comment by pixelzoom** [comment]
Taking a quick look at images/. Your home and nav icons could be created programmatically.  And many of your non-`world-temp*` images seems to be much bigger than necessary.  Anything bigger than 1.5x what you need is liable to end up with with scaling artifacts.

world-temperatures-monthy-averaged-2018-01.jpg and world-temperatures-monthy-averaged-2018-01.png appear to be duplicates.  Why do you need both formats?

So it appears as though there are opportunities to reduce footprint here.

**Comment by pixelzoom** [comment]
Taking a quick look at images/ ... 

* Your home and nav icons could be created programmatically. 
* Many of your non-`world-temp*` images seems to be much bigger than necessary.  Anything bigger than 1.5x what you need is liable to end up with with scaling artifacts.
* world-temperatures-monthy-averaged-2018-01.jpg and world-temperatures-monthy-averaged-2018-01.png appear to be duplicates.  Why do you need both formats?

So it appears as though there are opportunities to reduce footprint here.

**Comment by ariel-phet** [comment]
@jbphet - I agree this is getting to the "ball park", but I would investigate what @pixelzoom suggests above. Keeping our sims "lean" will help out with the app and such, and as we add sound files things will grow larger. If the suggestions above help and are not too much effort, I think they are worth it. I would call it done after that.

**Comment by jbphet** [comment]
I resized a bunch of the images to about as small as they can go without starting to look bad, at least on my system.  I also removed unused images (though I don't think they affected the build size).  The build size is currently at 3692k, which is very close to the same as EFAC.  I think this is about as good as we are going to get without a significant time investment.  @ariel-phet - okay with you if we call this good?
